exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,17 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if(mark >= 80){
+    return "A";
+  } else if (mark < 80 && mark > 60) {
+    return "B";
+  } else if (mark <= 60 && mark >50) {
```

Comment:
Your work here is definitely correct, but you can make it shorter by removing some code, for example you can remove  (mark < 80 && ) from second if, as the compiler will come here only when mark < 80. your code can be:
if(mark >= 80){
    return "A";
  } else if (mark > 60) {
    return "B";
  } else if (mark >50) {

Reply:
Thank you very much for your suggestion. I will amend my code.